utils/build-script-impl

```diff
@@ -2635,10 +2648,6 @@ for host in "${ALL_HOSTS[@]}"; do
                   continue
                 ;;
                 *)
-                  cmake_options=(
-                    ${cmake_options[@]}
-                    -DCMAKE_Swift_COMPILER:PATH="$(build_directory_bin ${LOCAL_HOST} swift)/swiftc"
-                  )
```

Comment:
Updated this pull for two swiftc invocations that I had missed: lldb above and removing this one that's unused, as `cmake_options` isn't invoked by the test commands. Since this flag was never used, just remove it.